src/modules/item/reducer.ts

```diff
@@ -124,6 +125,7 @@ export const INITIAL_STATE: ItemState = {
   data: {},
   rarities: [],
   loading: [],
+  hasUserOrphanItems: false,
```

Comment:
Quesdtion here! should we initialize this as `undefined` to avoid race conditions? if not, how we know if the user doesn't have orphans or if the request is still on flight?

Reply:
hey @juanmahidalgo -- good catch! 

currently, the value is set to `true` or keep `false` when the action `FETCH_ITEMS_SUCCESS` is executed, but it's required to trigger in a few components the `FETCH_ITEMS_REQUEST`. I'll update a little bit the logic.